src/registry/middleware/index.ts

```diff
@@ -41,7 +46,7 @@ module.exports.bind = function(app, options) {
   }
 
   if (options.local) {
-    app.use(errorhandler({ dumpExceptions: true, showStack: true }));
+    app.use(errorhandler());
```

Comment:
does `errorhandler()` have those previous settings as defaults?

Reply:
typescript complained those types don't exist. I checked the source code, those options dont exist